<launch>
  <arg name="image_topic" default="/kinect2/hd/image_color"/>
  <arg name="time_span" default="3"/>
  <arg name="save_directory" default="/home/gisen/ros/data/saved_image/"/>

  <node pkg="tabletop_detector" type="save_image.py" name="save_image"
        output="screen">
    <param name="image_topic" value="$(arg image_topic)" />
    <param name="time_span" value="$(arg time_span)" />
    <param name="save_directory" value="$(arg save_directory)" />
  </node>
</launch>
